### ISSUE TITLE
Wiki/documentation?

### ISSUE THREAD

**Issue by Frenzie** [XCMer/fuzzy-window-switcher#4]
The readme here does a good-enough job explaining things, but I made some notes while getting this to work in Debian Wheezy and I have no idea where to put them.
# Debian Wheezy

Debian doesn't currently come with keybinder-3.0, so to backport I followed a slight variation on the normal backporting procedure: I used Ubuntu 13.04 as my source instead of e.g. Debian testing or unstable.

```
# From http://packages.ubuntu.com/source/raring/keybinder-3.0
dget http://archive.ubuntu.com/ubuntu/pool/universe/k/keybinder-3.0/keybinder-3.0_0.3.0-0ubuntu1.dsc
dpkg-source -x keybinder-3.0_0.3.0-0ubuntu1.dsc 
cd keybinder-3.0-0.3.0/
# Here I typically add ~frenzie to my own packages.
debchange -R
# Check if you're missing something.
dpkg-checkbuilddeps
# I had to install these. You might need more.
sudo apt-get install libgirepository1.0-dev gobject-introspection
# And build the package.
dpkg-buildpackage -us -uc
cd ..
# We're done, install.
sudo dpkg -i *.deb

# We also need the backported python-configparser for Python 2.7. For how to add backports to your repositories see http://backports.debian.org/
sudo apt-get install python-configparser
```

Also, ConfigParser needs to be renamed to configparser. So it becomes import configparser and  configparser.ConfigParser

Before it'll work, the ~/.fuzzy-windows file is essential, otherwise it complains about section [Default] missing. (Okay, these last two are actually a bug reports too, but that aside.)

If I try to run it with python3, it actually complains about a bug in the built-in configparser.py, so for now it only runs with python (=2.7): in Wheezy that means python fuzzy-windows or simply ./fuzzy-windows

**Comment by XCMer** [comment]
I'll surely make some arrangements for a proper documentation.

The ConfigParser has been renamed to configparser in Python 3. The second error about the config file (~/.fuzzy-windows) has been fixed in master. The program is quite volatile for now. I'll get to a better release cycle once all the errors are ironed out.

**Comment by Frenzie** [comment]
On Debian Wheezy there only seems to be configparser in lowercase, whether for 2.6/2.7 or 3.2. A quick search suggests e.g. http://stackoverflow.com/questions/5847934/how-to-check-if-python-module-exists-and-can-be-imported could be used to maintain compatibility with both.

**Comment by XCMer** [comment]
I've just added a wiki. It should be a better and more organized document compared to the README. I'll update the README soon to reference the wiki instead. I've added your instructions to the installation page. Can you check it out? https://github.com/XCMer/fuzzy-window-switcher/wiki/Installation

**Comment by Frenzie** [comment]
Should be fine for now. Maybe a link could be added to http://wiki.debian.org/SimpleBackportCreation or some extra info here and there to e.g. explain that "The tilde ~ makes the package inferior in version, which should allow a proper package upgrade when you upgrade to the next debian release." In the end these are just some quick notes I jotted down while figuring out what I needed to make it work.

But I just noticed I have write access, so I'll just add such improvements myself. :)
